gns/render_rollout.py

```diff
@@ -38,13 +9,15 @@
 import os
 from pyevtk.hl import pointsToVTK
 
-flags.DEFINE_string("rollout_path", None, help="Path to rollout pickle file.")
+flags.DEFINE_string("rollout_dir", None, help="Directory where rollout.pkl are located")
+flags.DEFINE_string("rollout_name", None, help="Name of rollout `.pkl` file")
 flags.DEFINE_integer("step_stride", 3, help="Stride of steps to skip.")
-flags.DEFINE_boolean("block_on_show", True, help="For test purposes.")
+flags.DEFINE_enum("output_mode", "gif", ["both", "gif", "vtk"], help="Type of render output")
```

Comment:
don't use `both`, can we pass `--output-mode=gif --output-mode=vtk`?

Reply:
removed `both`.